README.rst

```diff
@@ -34,6 +34,10 @@ For Fedora and RHEL-derivatives, the following command will ensure that the requ
 
     $ sudo yum install gcc libffi-devel python-devel
 
+Alternatives
+============
+While bcrypt remains a good choice for password storage depending on your specific use case you may also want to consider using scrypt (either via `standard library`_ or a pypi package) or argon2id via `argon2_cffi`_.
```

Comment:
Is there a reason not to just link to cryptography for the "pypi package"?

Reply:
None other than that I literally forgot we support scrypt.